File: dziennik-elektroniczny-frontend/src/app/model/day-of-week.ts
import {Subject} from './subject';
import {HourLesson} from './hour-lesson';
import {Class} from './class';
import {Teacher} from './teacher';

export class DayOfWeek {

  dayOfWeekId: string;
  name: string;

  constructor(_name: string, _id?: string) {
    this.dayOfWeekId = _id;
    this.name = _name;
  }

}

File: dziennik-elektroniczny-backend/src/main/java/pl/karolmic/model/DayOfWeek.java
package pl.karolmic.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.OneToMany;
import java.util.List;

@Entity
public class DayOfWeek {

    @Id
    @Column(name = "day_of_week_id")
    private Integer id;

    private String name;

    @OneToMany
    List<Lesson> lessons;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return "DayOfWeek{" +
                "DayOfWeekId='" + id + '\'' +
                ", name='" + name + '\'' +
                '}';
    }
}
